datastore/postgres/indexfiles.go

```diff
@@ -0,0 +1,130 @@
+package postgres
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promauto"
+	"github.com/quay/claircore"
+	"github.com/quay/claircore/indexer"
+	"github.com/quay/claircore/pkg/microbatch"
```

Comment:
I don't know if we should keep using `microbatch` -- I guess it keeps this inline with the other indexer stuff.